### PR TITLE
Rack Middleware for Tracking Page Views

### PR DESCRIPTION
**Pull request by satoryu**
This pull-request provides a tiny rack middleware which inserts a script code to a content for tracking page views with application insights JavaScript SDK.  
This is inspired from this document:
 * [Azure Application Insights for JavaScript web apps | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript?toc=/azure/azure-monitor/toc.json#add-the-sdk-script-to-your-app-or-web-pages)

### CODE REVIEW

File: lib/application_insights/rack/track_page_view.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+require 'rack'
+
+module ApplicationInsights
+  module Rack
+    class TrackPageView
```
Comment (SergeyKanzhelev) [inline lib/application_insights/rack/track_page_view.rb]: perhaps it can be named differently. Typically `TrackFoo` method results in sending item. Maybe "InjectJavaScriptTracking"?

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
I suggested another name. What do you think?

File: lib/application_insights/rack/track_page_view.rb (hunk 1 above)
Comment (satoryu) [reply lib/application_insights/rack/track_page_view.rb]: Thanks. As you pointed out, it was wrong name. so I renamed it you suggested to me.

**Review by SergeyKanzhelev — APPROVED** [review]

File: lib/application_insights.rb [hunk 2]
```diff
@@ -5,5 +5,6 @@
 module ApplicationInsights
   module Rack
     autoload :TrackRequest, "application_insights/rack/track_request"
+    autoload :TrackPageView, "application_insights/rack/track_page_view"
```
Comment (SergeyKanzhelev) [inline lib/application_insights.rb:8]: I realize after merge that this wasn't renamed. Can you send separate PR please?

Comment (satoryu) [reply lib/application_insights.rb:8]: oh, sorry. I'll send new PR later.
